### PR TITLE
Fix add logo functionality for Pillow v10

### LINKED ISSUE #47: Add logo incompat with Pillow v10
Due to this line

https://github.com/pytroll/pydecorate/blob/4d21d613fdd38beffcef3c7fd2dfb7e37f49c7cd/pydecorate/decorator_base.py#L357

(`Image.ANTIALIAS` [was removed](https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#constants).)

Exception:
```
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```

### PR DESCRIPTION
**Pull request by zmoon**
Resolves #47

### CODE REVIEW

**Comment by zmoon** [comment]
I tested this fix with Pillow v9 and it works with that version as well.

File: pydecorate/tests/test_decorator_agg.py [hunk 1]
```diff
@@ -66,6 +70,13 @@ def test_colorbar(tmp_path, orientation_func_name, align_func_name, clims):
     )
 
 
+def test_add_logo():
+    img = Image.fromarray(np.zeros((200, 200, 3), dtype=np.uint8))
+    dc = DecoratorAGG(img)
+    dc.add_logo(REPO / "logos/pytroll_light_big.png")
```
Comment (djhoese) [inline pydecorate/tests/test_decorator_agg.py:76]: Technically shouldn't this be:

```suggestion
    dc.add_logo(REPO / "logos" / "pytroll_light_big.png")
```

to handle non-"/" systems properly?

Comment (zmoon) [reply pydecorate/tests/test_decorator_agg.py:76]: I think pathlib will handle it correctly for the platform either way.

Comment (djhoese) [reply pydecorate/tests/test_decorator_agg.py]: Yeah you might we right. I committed it anyway. Thanks again.

**Comment by zmoon** [comment]
@djhoese it's not related, but, any chance you could clear this up for me? https://github.com/pytroll/pytroll.github.io/issues/33

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Setuptools SCM version.py file
 pydecorate/version.py
 
+# Files created by test run
+style_retention.png
+add_logo_pytroll.png
```
Comment (djhoese) [inline .gitignore:6]: Is the `style_retention.png` one created by your new test too?

The test should be updated to use `tmp_path` for where it writes files.

Comment (zmoon) [reply .gitignore:6]: And for logo one I guess I could check that the new file is not all black for an indication it worked.

Comment (djhoese) [reply .gitignore:6]: If you're not familiar, you could use the tmp_path pytest fixture: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Comment (djhoese) [reply .gitignore:6]: And yeah checking for a non-black image sounds good enough.

**Pull request merged**